pkg/kubelet/kubelet_pods.go

```diff
@@ -1156,7 +1157,7 @@ func (kl *Kubelet) RunInContainer(podFullName string, podUID types.UID, containe
 
 	var buffer bytes.Buffer
 	output := ioutils.WriteCloserWrapper(&buffer)
-	err = kl.runner.ExecInContainer(container.ID, cmd, nil, output, output, false, nil)
+	err = kl.runner.ExecInContainer(container.ID, cmd, nil, output, output, false, nil, 0)
```

Comment:
*Aside:* Do you know what `RunInContainer` is used for? I feel like this should probably have a timeout too, or maybe the method should just be deprecated.